test/services/content/content.test.js

```diff
@@ -63,12 +62,8 @@ describe('content service', function () {
 		}));
 });
 
-function writeResponseToDisk(requestOptions, response) {
-	return writeFile(requestToFilename(requestOptions), response);
-}
-
 function requestToFilename(requestOptions) {
-	const key = JSON.stringify(requestOptions.qs).replace(/([^ -~]|[\."<>\|\\\/\:\*\?])+/g, ''); // just ASCII characters, NTFS-safe
+	const key = JSON.stringify(requestOptions.qs).replace(/([^ -~]|[."<>|\\/:*?])+/g, ''); // just ASCII characters, NTFS-safe
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 126. Maximum allowed is 120.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5405009)